osu.Desktop/Program.cs

```diff
@@ -104,6 +110,25 @@ public static void Main(string[] args)
             }
         }
 
+        [SupportedOSPlatform("windows")]
+        private static void setupSquirrel()
+        {
+            SquirrelAwareApp.HandleEvents(onInitialInstall: (version, tools) =>
+            {
+                tools.CreateShortcutForThisExe();
+            }, onAppUninstall: (version, tools) =>
+            {
+                tools.RemoveShortcutForThisExe();
+                tools.RemoveUninstallerRegistryEntry();
+            }, onEveryRun: (version, tools, firstRun) =>
+            {
+                tools.SetProcessAppUserModelId();
+
+                if (firstRun)
+                    tools.CreateUninstallerRegistryEntry();
```

Comment:
Is there a particular reason why this is done on first run rather than on initial install?

Reply:
I have no idea why this is here. I must have been sleep-coding.